test/basiccontrollermap.c

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
This file should be added to `test/basiscontrollermap.c` and probably also `test/Makefile.in`.

Reply:
Oh sorry, I didn't want to add this test here. I was thinking to do it in a separate PR

Reply:
File removed!